File: Module 1 - Decision Making/Decision Making/if Statement/Challenge Problem/AirportSecurity.py
"""Airport Security
Conditions:
Valid Passport
Valid Visa
Security Clearance
If all conditions are satisfied:
Boarding Allowed
Otherwise:
Boarding Denied"""

validPass = True
validVisa = True
securityClear = True

if validPass:
    if validVisa:
        if securityClear:
            print("Boarding Allowed")
else:
    print("Boarding Denied")

# another Way using Logical Operator

if validPass and validVisa and securityClear:
    print("Boarding Allowed")
else:
    print("Boarding Denied")

File: Module 1 - Decision Making/Decision Making/if Statement/Challenge Problem/AirportSecurity.cpp
/*
Airport Security
Conditions:
Valid Passport
Valid Visa
Security Clearance
If all conditions are satisfied:
Boarding Allowed
Otherwise:
Boarding Denied
*/

# include<iostream>
    using namespace std;

    int main(){
        bool validPass = true;
        bool validVisa = true;
        bool securityClear = true;

        if (validPass){
            if(validVisa){
                if(securityClear){
                    cout <<"Boarding Allowed\n";
                }
            }
        }

        else{
            cout <<"Boarding Denied";
        }

        // Another way using And Operator
        
        if(validPass && validPass && securityClear){
            cout <<"Boarding Allowed\n";
        }
        else{
            cout <<"Boarding Denied";
        }
        return  0;
    }